src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -54,23 +59,6 @@ class EmojiPickerMenu extends Component {
         // Ref for emoji FlatList
         this.emojiList = undefined;
 
-        // If we're on Windows, don't display the flag emojis (the last category),
-        // since Windows doesn't support them
-        const flagHeaderIndex = _.findIndex(emojis, (emoji) => emoji.header && emoji.code === 'flags');
-        this.emojis =
-            getOperatingSystem() === CONST.OS.WINDOWS
-                ? EmojiUtils.mergeEmojisWithFrequentlyUsedEmojis(emojis.slice(0, flagHeaderIndex))
-                : EmojiUtils.mergeEmojisWithFrequentlyUsedEmojis(emojis);
-
-        // Get the header emojis along with the code, index and icon.
-        // index is the actual header index starting at the first emoji and counting each one
-        this.headerEmojis = EmojiUtils.getHeaderEmojis(this.emojis);
```

Comment:
Did you delete this assignment entirely? I see you are calculating it inside `ComponentDidUpdate` but there is no assignment to `this.headerEmojis` there. I see it's used in CategoryShortcutBar, and because of that it's not visible now

Reply:
after: 
<img width="416" alt="image" src="https://github.com/Expensify/App/assets/61146594/3dae9835-fab3-4e9f-a026-cebaf888be30">

before:
<img width="413" alt="image" src="https://github.com/Expensify/App/assets/61146594/6cd4b43c-1d08-4087-bf87-96c08cdc46af">